project/threads/templates/threads/static_templates/landing.html

```diff
@@ -94,14 +92,13 @@ <h5 class="content-subtitle">Democratically Contributed Media</h5>
                         <i class="large material-icons">web</i>
                     </div>
                     <div class="col s12 m6 pull-m6">
-                        <h5 class="content-subtitle">Personalized Policy Feed</h5>
+                        <h5 class="content-subtitle">{% trans 'Personalized Policy Feed ' %}</h5>
                         <p class="content-text">
-                            CiviWiki intelligently personalizes users' feed in two meaningful ways.
+                            CiviWiki {% trans "intelligently personalizes users' feed in two meaningful ways. " %}
                             <br>
-                            <br> First, the issues promoted to users' feed will be personalized to the user's expressed interests, and the timeliness of the issue.
+                            <br> {% trans "First, the issues promoted to users' feed will be personalized to the user's expressed interests, and the timeliness of the issue." %}
                             <br>
-                            <br> Second, the structure of the issue topics break policy positions into bite-sized contentions we call Civies. Each Civi is logically related to the rest of the topic. Based on the user's support, opposition, or neutrality
-                            to each Civi, CiviWiki promotes different relevant content.
+                            <br> {% trans "Second, the structure of the issue topics break policy positions into bite-sized contentions we call Civies. Each Civi is logically related to the rest of the topic. Based on the user's support, opposition, or neutrality to each Civi, CiviWiki promotes different relevant content. "%}
```

Comment:
Trailing whitespace in translation string

```suggestion
                            <br> {% trans "Second, the structure of the issue topics break policy positions into bite-sized contentions we call Civies. Each Civi is logically related to the rest of the topic. Based on the user's support, opposition, or neutrality to each Civi, CiviWiki promotes different relevant content."%}
```